### PR TITLE
perf: better performance

### LINKED ISSUE #81: Page load slowly
Image

### PR DESCRIPTION
**Pull request by SiongSng**
Co-authored-by: 猴子 <a3510377@users.noreply.github.com>

# Description

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by MKMonkeyCat** [comment]
...手殘案到..

File: package.json [hunk 1]
```diff
@@ -37,7 +39,6 @@
     "eslint-plugin-vue": "9.0.1",
     "postcss-html": "1.4.1",
     "postcss-scss": "4.0.4",
-    "happy-dom": "4.1.0",
```
Comment (SiongSng) [inline package.json]: why remove happy-dom?

**Review by SiongSng — COMMENTED** [review]
LGTM (self review)

File: package.json (hunk 1 above)
Comment (banahaker) [reply package.json]: cause it's not used

File: .editorconfig [hunk 2]
```diff
@@ -7,6 +7,8 @@ indent_style = space # indent style / 縮排格式 ( tab | space )
 indent_size = 2 # indent size / 缩排大小
 trim_trailing_whitespace = true # trim trailing whitespace / 去除行首空白
 insert_final_newline = true # insert final newline  / 於文件尾插入空行
+end_of_line = lf # end of line / 文件行尾格式 ( lf | crlf )
```
Comment (banahaker) [inline .editorconfig]: 註解確定要中英夾雜了嗎

File: src/scss/global.scss [hunk 3]
```diff
@@ -1,12 +1,3 @@
-// Font - Roboto
```
Comment (banahaker) [inline src/scss/global.scss:1]: why delete those

**Review by banahaker — CHANGES_REQUESTED** [review]
Not bad but something wrong

File: src/scss/global.scss (hunk 3 above)
Comment (SiongSng) [reply src/scss/global.scss:1]: 加快載入時間，反正系統有字體除非要做特殊的文宣

**Review by banahaker — APPROVED** [review]
great

**Comment by banahaker** [comment]
may not has big problem

**Pull request merged**